Resources/Public/Javascript/PageView/FulltextControl.js

```diff
@@ -226,7 +226,7 @@ var dlfViewerFullTextControl = function(map, image, fulltextUrl) {
 
                         if (targetElem.length > 0 && !targetElem.hasClass('highlight')) {
                             targetElem.addClass('highlight');
-                            $('#tx-dlf-fulltextselection').scrollTo(targetElem, 50);
+                            setTimeout(this.scrollToText, 4000, targetElem);
```

Comment:
Timeout should be no longer than 1 second.

Reply:
Done